web/api/web_api_v1.c

```diff
@@ -212,6 +212,43 @@ inline int web_client_api_request_v1_alarms(RRDHOST *host, struct web_client *w,
     return 200;
 }
 
+#define CHECK_HTTP400(x) { if(!(x)) { error("CHECK_HTTP400(%s)", #x); return 400; } }
```

Comment:
We had a PR this week that changed web/server/web_client.h creating preprocessors for HTTP error messages, and this PR was already merged, please, use HTTP_RESP_BAD_REQUEST instead 400 here.